animal_crossing.py

```diff
@@ -83,9 +164,13 @@ def _stalk_set_sell_price(cmd, price):
 	with db:
 		cur = db.execute('SELECT week_index, price FROM sell_price WHERE user_id = ? AND week_local = ?',
 				(user_id, str(sunday),))
-		week_price_rows = cur.fetchall()
+		week_sell_rows = cur.fetchall()
+		cur = db.execute('SELECT price FROM buy_price WHERE user_id = ? AND week_local = ?',
+				(user_id, str(sunday),))
+		week_buy_row = cur.fetchone()
 	week_prices = [None] * 13
-	for row in week_price_rows:
+	week_prices[0] = week_buy_row['price']
```

Comment:
```suggestion
	if week_buy_row: 
		week_prices[0] = week_buy_row['price']
```
I think you need to handle the case where they haven't given a buy price